content/scoring/index.md

```diff
@@ -1,9 +1,17 @@
 ---
-title: Scoring
+linkTitle: -> We Made Worlds! <-
+title: How We did at worlds
 date: 2024-04-04T10:29:00-04:00
-draft: true
-hideFooter: True
-ShowWordCount: true
-ShowPostNavLinks: true
-ShowReadingTime: true
+draft: false
```

Comment:
Are you sure this page should be `draft: false` ?  we haven't made world yet

Reply:
sorry, was using it for debugging. ill fix in a min

Reply:
No biggie.